src/ripple/overlay/PeerSet.h

```diff
@@ -60,22 +75,14 @@ class PeerSet
 
     bool isActive ();
 
+    // ???
```

Comment:
I'm open to suggestions on what should go here. I don't know what the function does.